tests/Aspire.Pomelo.EntityFrameworkCore.MySql.Tests/AspireEFMySqlExtensionsTests.cs

```diff
@@ -247,6 +247,25 @@ public void CommandTimeoutFromSettingsWinsOverOthers(bool useSettings)
 #pragma warning restore EF1001 // Internal EF Core API usage.
     }
 
+    /// <summary>
+    /// Verifies that a DbContext can be registered with a specified context service (interface).
+    /// </summary>
+    [Fact]
+    public void CanSpecifyContextService()
+    {
+        var builder = Host.CreateEmptyApplicationBuilder(null);
+        builder.Configuration.AddInMemoryCollection([
+            new KeyValuePair<string, string?>("ConnectionStrings:mysql", ConnectionString),
+        ]);
+
+        builder.AddMySqlDbContext<ITestDbContext, TestDbContext>("mysql");
```

Comment:
You can try it, but now I am not sure that it'll work. That test shouldn't be connecting to the database at all, you might need to debug it to see why that's happening. It might be passing locally just because it successfully connects to the database.

Reply:
Unfortunately I'm not sure where to go from here. I only needed this for Postgres, and all of the tests for the other DB providers work.

If I stop Docker, I get 14 test failures for MySQL with:

```
Docker is either not running or misconfigured. Please ensure that Docker is running and that the endpoint is properly configured. You can customize your configuration using either the environment variables or the ~/.testcontainers.properties file. For more information, visit:
https://dotnet.testcontainers.org/custom_configuration/
```

If I start Docker, the tests all pass locally.

Reply:
If I'm reading the failed actions correctly, these same tests succeed on Linux. This is only failing on Windows, which may explain why I cannot reproduce it. I am not on Windows, nor do I have access to a Windows device. @AndriySvyryd @sebastienros are you able to run this failing test on a Windows machine?

Reply:
I took one more stab based on existing tests. All passing now.